docs/internal-documentation/distrib-testing/Ubuntu-build-environment.md

```diff
@@ -31,9 +31,13 @@ needs for testing and releasing Zonemaster:
 3. Install prerequisites (binaries)
 
    ```sh
-   sudo apt-get install cpanminus gettext autoconf automake build-essential libdevel-checklib-perl libmodule-install-xsutil-perl libssl-dev libidn11-dev libtool
+   sudo apt-get install cpanminus gettext autoconf automake build-essential libdevel-checklib-perl libmodule-install-xsutil-perl libssl-dev libidn2-dev libldns-dev” libtool
```

Comment:
There is a small typo right after “libldns-dev”.

Reply:
Small, but significant. Fixed.